06.wc/wc.rb

```diff
@@ -13,62 +13,54 @@ def parse_options
   options
 end
 
-def bytesize(text)
-  bytesize = []
-  text.each { |b| bytesize << b.bytesize }
-  bytesize.sum
+def count_bytesize(lines)
+  lines.sum(&:bytesize)
 end
 
-def line(text)
-  line = []
-  text.each { |l| line << l.scan(/\n/).size }
-  line.sum
+def count_line(lines)
+  lines.sum { |line| line.scan(/\n/).size }
 end
 
-def words(text)
-  word = []
-  text.each { |w| word << w.scan(/\S\s/).size }
-  word.sum
+def count_words(lines)
+  lines.sum { |line| line.split(/\s+/).size }
 end
 
+def no_options(options)
```

Comment:
```suggestion
def no_options?(options)
```

のように?を付けるとRubyらしくなりますね。

Reply:
なるほど、bool値の判定メソッドは?を付けるのですね。
no_options?に名称を修正しました。